index.js

```diff
@@ -73,9 +74,11 @@ async function main(opts) {
         });
       }
 
+      const normalized = new OASNormalize.default(oas);
+
       out.specs.push({
         fileName,
-        oas,
+        oas: JSON.stringify(await normalized.bundle()),
```

Comment:
can this arg ever be `undefined`? if so, trouble

Reply:
Dont believe so, but happy to write a test for that case if it comes to that!

Reply:
Added test for the empty state here: b533445fb9e3d45d6d741300461844548c1104f7